universal-application-tool-0.0.1/app/models/Version.java

```diff
@@ -0,0 +1,65 @@
+package models;
+
+import com.google.common.collect.ImmutableList;
+import io.ebean.annotation.UpdatedTimestamp;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import javax.persistence.Entity;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import play.data.validation.Constraints;
+
+@Entity
+@Table(name = "versions")
+public class Version extends BaseModel {
```

Comment:
suggestion (blocking): add javadoc for this class

Reply:
Done.